com.unity.netcode.gameobjects/Runtime/Components/NetworkAnimator.cs

```diff
@@ -605,6 +758,14 @@ protected virtual void Awake()
                 return;
             }
 
+            foreach (var parameterEntry in AnimatorParameterEntries.ParameterEntries)
+            {
+                if (!AnimatorParameterEntryTable.ContainsKey(parameterEntry.NameHash))
+                {
+                    AnimatorParameterEntryTable.Add(parameterEntry.NameHash, parameterEntry);
+                }
```

Comment:
nit: `TryAdd` will get the same functionality and will only hash the `NameHash` once rather than twice.

```suggestion
                AnimatorParameterEntryTable.TryAdd(parameterEntry.NameHash, parameterEntry);
```